db/migrate/20180605150932_deprecate_satellite_templates.rb

```diff
@@ -30,7 +30,7 @@ def remove_old_audits(destination)
   end
 
   def copy_os_default_associations(source, destination)
-    source.os_default_templates.update_all(provisioning_template_id: destination.id)
+    source.os_default_templates.update(provisioning_template_id: destination.id)
```

Comment:
I do not think these two are equivalent. can't we keep that cop disabled for this file instead?

Reply:
we can, so i disabled it in recent change.